DistributionSystemApi/DistributionSystemApi/ConfigurationExtensions/DatabaseConfigurationExtension.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DistributionSystemApi.Data;
+using Microsoft.EntityFrameworkCore;
+
+namespace DistributionSystemApi.ConfigurationExtensions
+{
+    public static class DatabaseConfigurationExtension
+    {
+        public static void ConfigureServices(this IServiceCollection services, IConfiguration configuration)
```

Comment:
those extensions should return IServiceCollection like microsofts configuration extensions do

Reply:
Also its a bit strange name